### PR TITLE
#7937 Remove usage from ToStringBuilder

### LINKED ISSUE #7937: Remove usage from `org.apache.commons.lang3.builder.ToStringBuilder`

### PR DESCRIPTION
**Pull request by timo0**
Fix #7937

### CODE REVIEW

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/events/BaseEventHashedData.java [hunk 1]
```diff
@@ -297,17 +295,17 @@ public int hashCode() {
...
-                .append("selfParentHash", CommonUtils.hex(valueOrNull(selfParentHash), TO_STRING_BYTE_ARRAY_LENGTH))
-                .append("otherParentHash", CommonUtils.hex(valueOrNull(otherParentHash), TO_STRING_BYTE_ARRAY_LENGTH))
-                .append("timeCreated", timeCreated)
-                .append("transactions size", transactions == null ? "null" : transactions.length)
-                .append("hash", CommonUtils.hex(valueOrNull(getHash()), TO_STRING_BYTE_ARRAY_LENGTH))
-                .toString();
+        return "BaseEventHashedData{" + "softwareVersion="
+                + softwareVersion + ", creatorId="
+                + creatorId + ", selfParentGen="
+                + CommonUtils.hex(valueOrNull(selfParentHash), TO_STRING_BYTE_ARRAY_LENGTH) + ", otherParentGen="
```
[the remark this answers is not in the document]
Comment (lpetrovic05) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/events/BaseEventHashedData.java:301]: this can be really useful information when debugging. I would say we leave it for now, events need an overhaul that I hope is coming in the near future

**Review by lpetrovic05 — DISMISSED** [review]

**Review by hendrikebbers — APPROVED** [review]

**Review by imalygin — APPROVED** [review]

File: platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java [hunk 2]
```diff
@@ -62,19 +59,20 @@ public record BenchmarkConfig(
...
-                .append("printHistogram", printHistogram)
-                .append("csvOutputFolder", csvOutputFolder)
-                .append("csvMetricsFileName", csvMetricsFileName)
-                .append("csvMetricNamesFileName", csvMetricNamesFileName)
-                .append("csvWriteFrequency", csvWriteFrequency)
-                .append("csvAppend", csvAppend)
-                .append("deviceName", deviceName)
-                .toString();
+        return "BenchmarkConfig{" + "benchmarkData='"
+                + benchmarkData + '\'' + ", saveDataDirectory="
```
Comment (alittley) [inline platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java:66]: The layout of this print is hard to follow, with the member name being on a different line from the value. This comment also applies to the other files in this PR, where this formatting is used

File: platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java [hunk 3]
```diff
@@ -62,19 +59,20 @@ public record BenchmarkConfig(
...
+                + saveDataDirectory + ", verifyResult="
+                + verifyResult + ", enableSnapshots="
+                + enableSnapshots + ", printHistogram="
+                + printHistogram + ", csvOutputFolder='"
+                + csvOutputFolder + '\'' + ", csvMetricsFileName='"
```
Comment (alittley) [inline platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java:71]: Are the single quotes around some of the values necessary? It appears that this is the only `toString` method which has these

File: platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java [hunk 4]
```diff
@@ -62,19 +59,20 @@ public record BenchmarkConfig(
...
+                + csvMetricsFileName + '\'' + ", csvMetricNamesFileName='"
+                + csvMetricNamesFileName + '\'' + ", csvWriteFrequency="
+                + csvWriteFrequency + ", csvAppend="
+                + csvAppend + ", deviceName='"
+                + deviceName + '\'' + '}';
```
Comment (alittley) [inline platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java:76]: If we want to remove the dependency on `ToStringBuilder`, IMO we should write a new method that creates a consistent `toString` format.

Trying to manually add equals, comma, and brackets is a recipe for inconsistency, as well as time wasted with incorrect formatting during PR review

**Comment by timo0** [comment]
Switched to own implementation in https://github.com/hashgraph/hedera-services/pull/8001
